apps/rush-lib/src/cli/actions/PublishAction.ts

```diff
@@ -173,7 +173,7 @@ export class PublishAction extends BaseRushAction {
    * Executes the publish action, which will read change request files, apply changes to package.jsons,
    */
   protected run(): Promise<void> {
-    if (!GitPolicy.getUserEmail(this.rushConfiguration, true)) {
+    if (!GitEmailPolicy.getUserEmail(this.rushConfiguration)) {
       process.exit(1);
```

Comment:
> process.exit(1); [](start = 5, length = 17)

It seems really strange that the contract for `getUserEmail()`  would be that if it returns an empty string, then the caller is supposed to silently exit the application because detailed error information was already printed to the console.